Jenkinsfile

```diff
@@ -1,3 +0,0 @@
-node {
```

Comment:
If you want to remove dmake CI builds, you should also remove the dmake.yml then (it is currently broken: it refers to `install_virtualenv.sh` which was removed in this PR).